src/BundledCodeBuilder.js

```diff
@@ -21,11 +21,22 @@ function bundledCode(serializedFeaturePath, isCyDashboard, cliOptions){
     // }
     //Run tests
     codeAsStr += `runTest(featureObj, ${cliOptions.routeCount});`;
+    
+    // normalize the arguments of CommonJS require() if the OS has '\' as path separator
+    codeAsStr = normalizeRequireFilePaths(codeAsStr)
+    
     return codeAsStr;
 }
 
 function requireInBrowser(moduleName){
     return "require('" + path.join( __dirname, moduleName) +"');"
 }
 
+function normalizeRequireFilePaths(fileContent) {
+    // if an OS is used that has '\' instead of '/' as path separator, then we need to
+    // replace the '\' with a '/' for the arguments of CommonJS require()
+    const backslashInRequirePathPattern = /(?<=require\(\'.*)\\(?=.*\'\))/g
+    return fileContent.replaceAll(backslashInRequirePathPattern, '/');
```

Comment:
In this code, we are replacing without checking what path separator OS supports. In my understanding, it will cause issue when using the code on Linux.

Reply:
Yes, you are right, thank you for pointing this out. This will cause issues when the file paths on an OS that does not use a backslash as path separator employ the backslash as an escape character. I will update the pull request with a fix.

The library is currently broken on any OS that does not use the forward slash as path separator though, since the NodeJS require() function only accepts forward slashes. To quote [the NodeJS documentation](https://nodejs.org/api/modules.html#requireid): 

> The relative paths of POSIX style are resolved in an OS independent fashion, meaning that the examples above will work on Windows in the same way they would on Unix systems.

POSIX style path implies using forward slashes in the file paths. I will update to pull request with a check which path separator the host uses before replacing.